frontend/src/routers/AppRouter/AppRouter.tsx

```diff
@@ -67,9 +70,35 @@ export const ABOUT_YOU_ROUTE = '/about_you'
 export const INVITE_TEAM_ROUTE = '/invite_team'
 export const SETUP_ROUTE = '/setup'
 
+const DebugRoutes: React.FC<React.PropsWithChildren> = ({ children }) => {
```

Comment:
this component does nothing in prod since `log` does nothing in prod. could be useful for debugging future react router bugs